apps/site/components/Icon/index.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable */
```

Comment:
please do not disable, generically, the linter

if you MUST disable
- disable specific `rules`
- disable specific `lines of code`